### DISCUSSION TITLE
Questions about the aws service plug-in

### DISCUSSION THREAD

**Discussion by gnzmzn321** [cloudforet-io/community#128] [Q&A]
Hello, I have a question!
I linked it with the aws account after installation!
AWS Personal Health Dashboard Collector 와
AWS Trusted Advisor Collector Collector encountered an error
First of all, the PersonalHealthDashboard encountered a SubscriptionRequiredException error!
Should I upgrade the dashboard's plan and access the API for this part?
In the case of AWS Trusted Advisor Collector, an error called No message from plugin occurred.
I've confirmed that I collect it well in aws console.
![캡처](https://github.com/cloudforet-io/cloudforet/assets/104878970/54090d78-f9f8-41a4-bd22-a4f298e0dd9c)
![캡처2](https://github.com/cloudforet-io/cloudforet/assets/104878970/7626ef42-10ef-47f0-aed6-d631548d259e)

**Comment by jihyungSong** [comment]
Hi, 
The `SubscriptionRequiredException` error that occurs when you run the PHD (Personal Health Dashboard) Collector is an issue related to the support plan of your AWS account. To query PHD via API, your AWS account must be on a Business plan or higher.

This is also for Trusted Advisor. In order to use this API, you must have a Business plan or higher, too.

Below are the related pages for the Python SDK used to collect PHD and TA. 
It's worth noting that the Support API is only available for use with a Business Plan or higher.

https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/support.html

In the AWS Console, both PHD and TA can be checked even with a basic account. 
However, to use the API, it needs to be a Business Plan or higher.
